content/blog/backstage-weekly-28-adopting-backstage/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Backstage Weekly 28 - A guide to adopting Backstage
+date: '2021-05-31T21:00:00.0Z'
+description: Plus... a recap of community session 4 and a look at the new Backstage catalog UI.
+author:
+  name: David Tuite
+tags: ['newsletter']
+---
+
+Having talked to many companies who are adopting Backstage, one thing is clear... Backstage means different things to different people. Because Backstage is a flexible and highly customizable framework, it can be tricky to figure out how to get started with it.
+
+Clear direction and implementation guidance is part of the value we bring at [Roadie](https://roadie.io), but it's also a concern for the Backstage core team.
+
+To help companies orient themselves, Austin Lamon — Director of Product for Platform at Spotify — recently published a guide to adopting Backstage on the official Backstage blog.
+
+Austin's guide describes three key of a software engineer — create, manage and explore — and explains how Backstage can help with each job. It also provides headcount tailored recommendations on which job to target with your Backstage MVP.
```

Comment:
```suggestion
Austin's guide describes three key tasks of a software engineer — create, manage and explore — and explains how Backstage can help with each job. It also provides headcount tailored recommendations on which job to target with your Backstage MVP.
```

Reply:
or `jobs`

Reply:
I'll go with jobs since they're talking about the jobs to be done framework in the post.